can_messages.asciipb

```diff
@@ -1,9 +1,25 @@
 msg {
```

Comment:
Do you want to make an example.asciipb that looks like this and then make the can_messages.asciipb empty? I feel like since this is the filename of the file we will actually use for the car it shouldn't have test data in it.